src/martok/ImportGenerator.ts

```diff
@@ -10,8 +10,10 @@ import _ from "lodash";
 import { Martok } from "./Martok";
 
 export class ImportGenerator {
-  private readonly checker = this.martok.program.getTypeChecker();
-  public constructor(private readonly martok: Martok) {}
```

Comment:
This change shouldn't be necessary-- was it a lint warning or something?

Reply:
I was getting an issue where checker was undefined. Maybe not in this class, but in a few others. Since the `ts.Program` is created in the Martok class constructor I wanted to be extra sure that anything that used martok was getting initialized with the correct `ts.Program`.

Reply:
I'll revert, this probably isn't actually doing anything.